qa/L0_inferentia_perf_analyzer/test.sh

```diff
@@ -0,0 +1,134 @@
+echo "done setting up enviroment"
+
+REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+if [ "$#" -ge 1 ]; then
+    REPO_VERSION=$1
+fi
+
+CLIENT_LOG="./perf_analyzer.log"
+PERF_ANALYZER=/opt/tritonserver/qa/clients/perf_analyzer
+
+OUTPUT_JSONDATAFILE=${TEST_JSON_REPO}/validation.json
+NON_ALIGNED_OUTPUT_JSONDATAFILE=${TEST_JSON_REPO}/non_aligned_validation.json
+WRONG_OUTPUT_JSONDATAFILE=${TEST_JSON_REPO}/wrong_validation.json
+
+ERROR_STRING="error | Request count: 0 | : 0 infer/sec"
+
+SERVER=/opt/tritonserver/bin/tritonserver
+SERVER_LOG="./inference_server.log"
+source /opt/tritonserver/qa/common/util.sh
+TEST_TYPES="single multiple"
+
+# Setup models
+for TEST_TYPE in $TEST_TYPES; do
+    DATADIR="${TRITON_PATH}/models_${TEST_TYPE}"
+    rm -rf DATADIR
+done
+cd ${TRITON_PATH}
+python ${TEST_JSON_REPO}/simple-model.py
+python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
+    --model_type "pytorch" \
+    --triton_input INPUT__0,INT64,4 INPUT__1,INT64,4 \
+    --triton_output OUTPUT__0,INT64,4 OUTPUT__1,INT64,4 \
+    --compiled_model $PWD/add_sub_model.pt \
+    --triton_model_dir models_single/add-sub-1x4 --neuron_core_range 0:0
+
+cd ${TRITON_PATH}
+python ${TEST_JSON_REPO}/simple-model.py
+python ${TRITON_PATH}/python_backend/inferentia/scripts/gen_triton_model.py \
+    --model_type "pytorch" \
+    --triton_input INPUT__0,INT64,4 INPUT__1,INT64,4 \
+    --triton_output OUTPUT__0,INT64,4 OUTPUT__1,INT64,4 \
+    --compiled_model $PWD/add_sub_model.pt \
+    --triton_model_dir models_multiple/add-sub-1x4 \
+    --triton_model_instance_count 3 --neuron_core_range 0:7
+
+RET=0
+
+for TEST_TYPE in $TEST_TYPES; do
+    DATADIR="${TRITON_PATH}/models_${TEST_TYPE}"
+    SERVER_ARGS="--model-repository=${DATADIR} --log-verbose=1"
+    rm -f $SERVER_LOG $CLIENT_LOG
+
+    run_server
+    if [ "$SERVER_PID" == "0" ]; then
+        echo -e "\n***\n*** Failed to start $SERVER\n***"
+        cat $SERVER_LOG
+        exit 1
+    fi
+    set +e
+    $PERF_ANALYZER -v -m add-sub-1x4 --input-data=${NON_ALIGNED_OUTPUT_JSONDATAFILE} >$CLIENT_LOG 2>&1
+    if [ $? -eq 0 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+    if [ $(cat $CLIENT_LOG |  grep "The 'validation_data' field doesn't align with 'data' field in the json file" | wc -l) -eq 0 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+
+    $PERF_ANALYZER -v -m add-sub-1x4 --input-data=${WRONG_OUTPUT_JSONDATAFILE} >$CLIENT_LOG 2>&1
+    if [ $? -eq 0 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+    if [ $(cat $CLIENT_LOG |  grep "Output doesn't match expected output" | wc -l) -eq 0 ]; then
+        cat $CLIENT_LOG
+        echo -e "\n***\n*** Test Failed\n***"
+        RET=1
+    fi
+
+    $PERF_ANALYZER -v -m add-sub-1x4 --input-data=${OUTPUT_JSONDATAFILE} >$CLIENT_LOG 2>&1
```

Comment:
One small thing, Can you add a test case with a larger concurrency? For example 10? It will test the infrastructure under heavy load..